assets/javascripts/legal-summaries.js

```diff
@@ -0,0 +1,29 @@
+(function($) {
+
+  var summarise = function(){
```

Comment:
Very minor thing, but if you name this function, e.g.
```javascript
var summarise = function summarise() {
```

It gets a name when it's debugged, which is nice.